docs/tutorials/8_train_custom_env.ipynb

```diff
@@ -0,0 +1,494 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[download this notebook here](https://github.com/HumanCompatibleAI/imitation/blob/master/docs/tutorials/8_train_custom_env.ipynb)\n",
+    "# Train Behavior Cloning in a Custom Environment\n",
+    "\n",
+    "You can use `imitation` to train a policy in a custom environment.\n",
```

Comment:
This might give the impression that `imitation` only supports training policies in custom environments, and not training rewards, when in fact it supports both (although behavioral cloning can only learn policies).